src/inputs/button/button.jsx

```diff
@@ -330,22 +335,20 @@ class Button extends React.PureComponent {
             {
                 [classes.inverse]: inverse,
                 [classes.colorAlert]: !disable && color === 'alert',
-                [classes.colorAlternate]: !disable && color === 'alternate',
+                [classes.colorSecondary]: !disable && color === 'alternate',
                 [classes.colorDisable]: disable || color === 'disable',
                 [classes.colorInverse]: !disable && color === 'inverse',
-                [classes.colorLight]: !disable && color === 'light',
-                [classes.colorOutline]: !disable && color === 'outline',
                 [classes.colorPrimary]: !disable && (!color || color === 'primary'),
-                [classes.colorSecondary]: !disable && color === 'secondary',
                 [classes.colorSuccess]: !disable && color === 'success',
-                [classes.colorTransparent]: !disable && color === 'transparent',
                 [classes.colorWarning]: !disable && color === 'warning',
+                [classes.colorTransparent]: !disable && transparent,
```

Comment:
I think that we should be able to have a transparent button and for the button to be disabled.